packages/esm-patient-biometrics-app/webpack.config.js

```diff
@@ -1,80 +1,85 @@
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const { resolve } = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+const { StatsWriterPlugin } = require('webpack-stats-plugin');
 
-const { peerDependencies } = require("./package.json");
+const { peerDependencies } = require('./package.json');
 
+const filename = 'openmrs-esm-patient-biometrics-app.js';
 const cssLoader = {
-  loader: "css-loader",
+  loader: 'css-loader',
   options: {
     modules: {
-      localIdentName:
-        "esm-patient-biometrics__[name]__[local]___[hash:base64:5]"
-    }
-  }
+      localIdentName: 'esm-patient-biometrics__[name]__[local]___[hash:base64:5]',
+    },
+  },
 };
 
 module.exports = (env, argv = {}) => ({
-  entry: [
-    resolve(__dirname, "src/set-public-path.ts"),
-    resolve(__dirname, "src/index.ts")
-  ],
-  mode: argv.mode || "development",
+  entry: [resolve(__dirname, 'src/set-public-path.ts'), resolve(__dirname, 'src/index.ts')],
+  mode: argv.mode || 'development',
   output: {
-    filename: "openmrs-esm-patient-biometrics-app.js",
-    libraryTarget: "system",
-    path: resolve(__dirname, "dist"),
-    jsonpFunction: "webpackJsonp_openmrs_esm_patient_biometrics_app"
+    filename,
+    libraryTarget: 'system',
+    path: resolve(__dirname, 'dist'),
+    jsonpFunction: 'webpackJsonp_openmrs_esm_patient_biometrics_app',
   },
   module: {
     rules: [
       {
         parser: {
-          system: false
-        }
+          system: false,
+        },
       },
       {
         test: /\.m?(js|ts|tsx)$/,
         exclude: /(node_modules|bower_components)/,
         use: {
-          loader: "babel-loader"
-        }
+          loader: 'babel-loader',
+        },
       },
       {
         test: /\.css$/,
-        use: ["style-loader", cssLoader]
+        use: ['style-loader', cssLoader],
       },
       {
         test: /\.s[ac]ss$/i,
-        use: ["style-loader", cssLoader, "sass-loader"]
+        use: ['style-loader', cssLoader, 'sass-loader'],
       },
       {
         test: /\.(png|jpe?g|gif)$/i,
         use: [
           {
-            loader: "file-loader"
-          }
-        ]
-      }
-    ]
+            loader: 'file-loader',
+          },
+        ],
+      },
+    ],
   },
-  devtool: "sourcemap",
+  devtool: 'sourcemap',
   devServer: {
     headers: {
-      "Access-Control-Allow-Origin": "*"
+      'Access-Control-Allow-Origin': '*',
     },
-    disableHostCheck: true
+    disableHostCheck: true,
   },
   externals: Object.keys(peerDependencies),
   plugins: [
     new ForkTsCheckerWebpackPlugin(),
     new CleanWebpackPlugin(),
     new BundleAnalyzerPlugin({
-      analyzerMode: env && env.analyze ? "server" : "disabled"
-    })
+      analyzerMode: env && env.analyze ? 'server' : 'disabled',
+    }),
+    new StatsWriterPlugin({
```

Comment:
We could already add this to all webpack configurations.

Reply:
(that way independent of they provide online or offline components the MFs are prepared)